### PR TITLE
Added loading to contributors page

### PR DESCRIPTION
**Pull request by t-kashima**
## Issue

- https://github.com/DroidKaigi/conference-app-2017/issues/167

## Overview (Required)

- Show progress bar on loading contributors data

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java [hunk 1]
```diff
@@ -50,6 +61,15 @@ public void onClickContributor(String htmlUrl) {
         }
     }
 
+    @Bindable
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java]: I think it's not good to check by `contributors`. How about adding `setter` and `getter`?
Like this

```java
private int loadingVisibility;

@Bindable
public int getLoadingVisibility() {
    return loadingVisibility;
}

public void setLoadingVisibility(int visibility) {
    this.loadingVisibility = visibility;
    notifyPropertyChanged(BR.loadingVisibility);
}
```

```
       return contributorsRepository.findAll().map(contributors -> {
            this.contributors = contributors;
            setLoadingVisibility(View.VISIBLE);
```

Comment (t-kashima) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java]: Thank you for the review.😆 
That's good! I add a loadingVisibility variable.

**Comment by t-kashima** [comment]
I added a variable to switch the visibility😀

**Comment by konifar** [comment]
Good work! LGTM! Thank you for contribution! 👏

**Pull request merged**